share/shaders/FlowGlyphsSphere.frag

```diff
@@ -0,0 +1,57 @@
+#version 330 core
```

Comment:
Document. What does this program do?

Reply:
A bunch of these files that you requested documentation for are straightforward fragment shaders. These should be self-explanatory and thus none of the other (non raytracing) fragment shaders have documentation. Do you still want me to write documentation for these?

Reply:
Done